### PR TITLE
Fix the unit test failure issue.

### PR DESCRIPTION
**Pull request by buptkang**
This PR fix the unit test failure, which was caused by UWP appxbundle.

### CODE REVIEW

File: src/contentAPPXbundle.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class AppxBundleContent extends ContentBase {
 
         this.CheckManifestForAppx(manifestData, fileList);
         if(this.appxName) {
-            return await this.readAppx(fileList, tempDir);
```
Comment (izikl) [inline src/contentAPPXbundle.ts]: If I remember correctly, if the appx is found in the manifest then we can extract all the data from it, so there is no need to continue the flow.

Comment (buptkang) [reply src/contentAPPXbundle.ts]: If return here, properties such as build version will not be assigned which makes the unit test failed.

File: src/contentAPPXbundle.ts [hunk 2]
```diff
@@ -70,7 +70,8 @@ export class AppxBundleContent extends ContentBase {
     // FINDS THE APPX INSIDE THAT HAS *EVERYTHING* IMPORTANT
     private CheckManifestForAppx(manifestData: any, fileList: string[]) {
         if (manifestData.Bundle.Packages[0].Package) {
```
[the remark this answers is not in the document]
Comment (buptkang) [reply src/contentAPPXbundle.ts]: Rollback this change since it is related to the unit test failure issue.

File: src/contentAPPXbundle.ts (hunk 1 above)
Comment (izikl) [reply src/contentAPPXbundle.ts]: Ok removing the return means:
1. All the languages are taken from the appxbundle and not from the  inner appx. 
2. If icon wasn't extracted from the inner appex, we will try to extract it from the bundle. 

Both options makes sense to me. So let's keep it the way it is.

**Comment by buptkang** [comment]
We will further monitor this change.

**Pull request merged**